doc-content/enterprise-only/admin-and-config/managing-business-central-example-remote-git-repository-integration-ref.adoc

```diff
@@ -0,0 +1,143 @@
+----
+. Optional: To create a template `.gitremote` configuration file, run `git-push-2.1-SNAPSHOT.jar`:
++
+[source]
+----
+$ java -jar git-push-2.1-SNAPSHOT.jar
+----
++
+.Example template `.gitremote` configuration file
+[source]
+----
+#This is an auto generated template empty property file
+provider=GIT_HUB
+login=
+password=
+token=
+remoteGitUrl=https://api.github.com/
+useSSH=false
+ignore=.*demo.*, test.*
+githubOrg=OrgName
+gitlabGroup=Group/subgroup
+----
+
+. Modify the `.gitremote` configuration file parameters.
++
+.Example `.gitremote` parameters
+[cols="25%,75%", options="header"]
+|===
+|Parameter
+|Description
+
+|`provider`
+|The Git provider. Only two values are accepted: GIT_HUB and GIT_LAB. Required
+
+|`login`
+|The username for the Git provider. Required
+
+|`password`
+|A plain text password. Not required if a `token` is provided.
+
+|`token`
+|A generated token to replace the `username` and `password` based unsecured connection. Note: If this is not set a warning is displayed that you are using an unsecured connection. Not required if a `password` is provided. Note: GitLab only supports token authentication.
+
+|`remoteGitUrl`
+|A public provider URL or a locally hosted enterprise for any provider. Required. Note: The public GitHub URL should be the API URL. For example, api.github.com.
+
+|`useSSH`
+|Boolean to allow the SSH protocol to push changes to the remote repository. Optional. Default = false. Note: This parameter uses the local `~/.ssh/` directory to obtain the SSH configuration.
+
+|`ignore`
+|A comma separated regular expressions to ignore project names that match any of these expressions. Optional.
+
+|`githubOrg`
+|Defines the repository organization if GitHub is used as the provider. Optional.
+
+|`gitlabGroup`
+|Defines the repository group and subgroup if GitLab is used as the provider Optional.
+|===
+
+. Create a `post-commit` Git hook file in `_EAP_HOME_/hooks`:
++
+[source]
+----
+$ touch post-commit
+----
+. Set the permissions of the `post-commit` file to `755`:
++
+[source]
+----
+$ chmod 755 post-commit
+----
+. Add `#!/bin/bash` and code to execute `git-push-2.1-SNAPSHOT.jar` to the `post-commit` file:
++
+[source]
+----
+$ echo "#\!/bin/bash\njava -jar $APP_SERVER_HOME/hooks/git-push-2.1-SNAPSHOT.jar" > hooks/post-commit
+----
+
+. Start {CENTRAL} with the `-Dorg.uberfire.nio.git.hooks` environment variable set. For example:
++
+[source]
+----
+$ ./standalone.sh -c standalone-full.xml -Dorg.uberfire.nio.git.hooks=_EAP_HOME_/hooks
+----
+
+[NOTE]
+====
+To use post-commit Git hooks that execute Java code, you must use the following Java libraries:
+
+* https://www.eclipse.org/jgit/[JGit]: Used to interact with internal {CENTRAL} Git repositories.
+* http://github-api.kohsuke.org/[GitHub API for Java]: Used to communicate with GitHub.`t repositories.
+* http://github-api.kohsuke.org/[GitHub API for Java]: Used to communicate with GitHub.
```

Comment:
The double occurrence of  **GitHub API for Java** link.
Both the bullet points from the NOTE reflects same URL.
Not sure what is **`t repositories**? Is this a typo? 
Please correct me if I am wrong.

Reply:
Thanks, this is a pasting mistake.
Correcting..